walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -14,6 +14,52 @@ button {
   margin-top: 10px;
   width: 50%;
 }
+  
+.about-container .description p {
+  margin-left: 30px;
+}
+
+.work-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  margin-bottom: 10px;
+  margin-top: 100px;
+  width: 100%;
+}
+
+.jobs-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.jobs-container .work {
+  margin-right:auto;
+  margin-left: auto;
+}
+
+.work img {
+  max-width: 500px;
+  min-height: 300px;
+  min-width: 200px;
+}
+
+.projects-container {
+  display:flex;
```

Comment:
here too, again, set up an auto formatter, it's the best thing ever
if you use VSCode, just use Prettier, otherwise use any of the methods that Colm shared